GameV0Component.js

```diff
@@ -23,20 +25,34 @@ class GameV0Component extends EntityComponent {
   }
 
   think(dt) {
+    this.next_spawn_time -= dt;
+    if (this.next_spawn_time <= 0) {
+      this.queen_spawn_rate /= 1.2; // progressively spawn faster
+      this._spawnQueen();
+      this.next_spawn_time = this.queen_spawn_rate;
+    }
     this._timers = _.filter(this._timers, (timer) => {
       return timer(dt);
     });
+
   }
 
   componentDidMount() {
   }
 
+  _updateQueenRespawnTimers() {
+    this.queen_spawn_rate = initial_queen_spawn_rate / _.size(this._playerByID);
+    this.next_spawn_time = this.queen_spawn_rate;
+  }
+
   onPlayerAdded(player) {
```

Comment:
I think we should do bette rthan just resetting to zero when a player is added or removed

how about just keeping it as is, unless we added the first player (then start spawning) or removed the last player (then don't spawn anymore)